src/lib/receiver.js

```diff
@@ -1,15 +1,17 @@
 'use strict'
 
-const crypto = require('crypto')
 const uuid = require('uuid')
 const moment = require('moment')
 const Client = require('ilp-core').Client
 const EventEmitter = require('eventemitter2')
+const base64url = require('../utils/base64url')
+const cc = require('../utils/condition')
 const debug = require('debug')('ilp:receiver')
 const BigNumber = require('bignumber.js')
 const createHmacHelper = require('../utils/hmac').createHmacHelper
 
-const RECEIVER_ID_PREFIX = '~ipr.'
+const IPR_RECEIVER_ID_PREFIX = '~ipr.'
+const SSP_RECEIVER_ID_PREFIX = '~ssp.'
```

Comment:
I think it would be better to separate the IPR and SSP receivers into completely separate modules. Chances are you won't be using both at the same time and I think it could cause issues having the same receiver instance listening on two separate protocols.